app/testOnly/models/formsmodel/StartJourneyForm.scala

```diff
@@ -89,10 +89,12 @@ object StartJourneyForm {
   private val payeDebtTotalAmountKey: String = "payeDebtTotalAmount"
   private val vatDebtTotalAmountKey: String = "vatDebtTotalAmount"
   private val saDebtTotalAmountKey: String = "saDebtTotalAmount"
+  private val siaDebtTotalAmountKey: String = "siaDebtTotalAmount"
 
   private val payeTaxReferenceKey: String = "payeTaxReference"
   private val vatTaxReferenceKey: String = "vatTaxReference"
   private val saTaxReferenceKey: String = "saTaxReference"
+  private val siaTaxReferenceKey: String = "siaTaxReference"
```

Comment:
wouldn't this be `nino` now that you've changed the id in the template? You'd be able to tell if it's working by entering a NINO on the start page and checking the journey in mongo has the NINO you typed in once it's created